src/shorthands/JImage.jl

```diff
@@ -1,17 +1,17 @@
-function _JImage(pos, img, centering, clipargs, clipshape)
-    if clipshape != false
-        clipshape(clipargs...)
+function _JImage(pos, img, centering, shapeargs, shape)
+    if shape != false
+        shape(shapeargs...)
```

Comment:
One minor thing:
I think we should mention(in the docstring) that the shape should be a valid Luxor shape.
eg: rect, box, poly etc.

Reply:
"valid Luxor shape" I'm not aware of this concept. :) Do you mean a built-in Luxor function - eg one that constructs paths, or one that generates points for polygons...? (Not trying to be awkward, just I get worried in case I'm not providing the tools you folks need for your cool constructions!)

Reply:
>Do you mean a built-in Luxor function - eg one that constructs paths
Yeah! We need shape to be a valid luxor function, but a user may interpret it as entering a full shape name eg: rectangle instead of `rect`.

Reply:
We mean any kind of function either their own or more likely a Luxor drawing function. Mostly those which provide the clipping action (so yeah a drawing function 😂)

Reply:
OK, yes. Perhaps any function that has an `action` parameter/keyword...

Reply:
Another thing I want to mention is that `shape` should by typed to `Method`. 
eg: If there's a variable `circle = Object(...)`, and user passes circle as an argument, it will throw the `Objects of type Javis.Object are not callable` error and maybe use a try catch block to give out a better error message?